src/data/general-cs-quiz.ts

```diff
@@ -714,6 +714,15 @@ const generalCSQuiz = [
     Explanation: "SVG stands for Scalable Vector Graphics.",
     Link: "https://www.freecodecamp.org/news/use-svg-images-in-css-html/",
   },
+  {
+    Question: "Are JPEG and JPG sare the same?",
+    Answer: "Yes",
+    Distractor1: "May Be",
+    Distractor2: "No",
+    Distractor3: "Can't be said",
+    Explanation: "Yes - the only difference is that, traditionally, file extensions are only 3 characters long. '.jpg' is a shortened form of '.jpeg'.",
+    Link: "https://www.freecodecamp.org/news/image-file-types-picture-format-extensions-jpeg-gif-png-svg-tiff/",
+  },
 ];
```

Comment:
We want to try and avoid questions with only yes or no answers.

Let's go with this alternate question instead which better fits the computer science category. 
```
{
    Question: "What is Lossless compression?",
    Answer: "a compression algorithm that allows the original data to be perfectly reconstructed from the compressed data.",
    Distractor1: "a sorting algorithm",
    Distractor2: "a reverse algorithm",
    Distractor3: "an algorithm that converts wav files to mp3 files",
    Explanation: "Lossless compression is a compression algorithm that allows the original data to be perfectly reconstructed from the compressed data.",
    Link: "https://www.freecodecamp.org/news/image-file-types-picture-format-extensions-jpeg-gif-png-svg-tiff/",
  },
```